jobs/batch_jobs/cron_jobs.py

```diff
@@ -0,0 +1,222 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Jobs that are run by CRON scheduler."""
+
+from __future__ import absolute_import
+from __future__ import annotations
+from __future__ import unicode_literals
+
+from core.domain import search_services
+from core.domain import user_services
+from core.platform import models
+import feconf
+from jobs import base_jobs
+from jobs import job_utils
+from jobs.io import ndb_io
+from jobs.types import job_run_result
+
+import apache_beam as beam
+
+from typing import List, cast # isort:skip
+
+MYPY = False
+if MYPY: # pragma: no cover
+    from mypy_imports import datastore_services
+    from mypy_imports import exp_models
+    from mypy_imports import user_models
+
+
+(exp_models, user_models) = models.Registry.import_models(
+    [models.NAMES.exploration, models.NAMES.user])
+datastore_services = models.Registry.import_datastore_services()
+platform_search_services = models.Registry.import_search_services()
+
+
+class IndexExplorationsInSearch(base_jobs.JobBase):
+    """Job that indexes the explorations in Elastic Search."""
+
+    MAX_BATCH_SIZE = 1000
+
+    @staticmethod
+    def _index_exploration_summaries(
+            exp_summary_models: List[datastore_services.Model]
```

Comment:
Why not make this `List[exp_models.ExpSummaryModel]`?

Reply:
That doesn't work, I will be refactoring the Query stub so that we can do this.